package.json

```diff
@@ -122,9 +117,8 @@
     "last 2 iOS versions"
   ],
   "lint-staged": {
-    "components/*/package.json": [
-      "yarn lint:components",
-      "prettier-package-json --write"
+    "*.css": [
+      "stylelint --fix --quiet"
```

Comment:
I'm a little leery of this, just because we had some problems the last time. Let's move forward with this for now, but keep an eye on it for issues, please!